### PR TITLE
enh: Standardising Header Bar as of Open Event

### LINKED ISSUE #2092: Standard UI Buttons as done in Open Event
Currently, Buttons are too plain and don't indicate the `Semantic UI Button Feature`, As no item effect on them, Implement them as done in Open Event

### PR DESCRIPTION
**Pull request by kushthedude**
Fixes #2092

### CODE REVIEW

File: frontend/app/templates/components/nav-bar.hbs [hunk 1]
```diff
@@ -1,44 +1,43 @@
...
-            {{/link-to}}
-        {{else}}
-            {{#link-to 'login' class='item ui orange text create' click=(action 'loginToContinue')}}
-                CREATE BADGES
-            {{/link-to}}
-        {{/if}}
-    </div>
-</div>
\ No newline at end of file
+<div class="mobile hidden row">
```
Comment (abishekvashok) [inline frontend/app/templates/components/nav-bar.hbs]: Please conform with the intendation of 4 spaces used throughout the source code.

Comment (kushthedude) [reply frontend/app/templates/components/nav-bar.hbs]: I just missed a thing , It will hide the complete nav-bar in `Mobile View`

Comment (abishekvashok) [reply frontend/app/templates/components/nav-bar.hbs]: I spoke about intendation, anyway if it's WIP pls add the necessary changes to the pr title.

Comment (abishekvashok) [reply frontend/app/templates/components/nav-bar.hbs]: I mentioned about intendation, anyway if it's WIP please mark it as WIP.

**Comment by kushthedude** [comment]
> Please match the intendation with 4 spaces instead of 6

Where did you see the indents of 6 ?

**Comment by rajvaibhavdubey** [comment]
> Please match the intendation with 4 spaces instead of 6

@abishekvashok Indentation is fine, matches the source :+1:

**Pull request merged**